server/sonar-search/src/main/java/org/sonar/search/EsSettings.java

```diff
@@ -49,8 +52,14 @@
 
   EsSettings(Props props) {
     this.props = props;
-    // name of ES cluster must always be set, even if clustering of SQ is disabled
-    this.clusterName = props.nonNullValue(ProcessProperties.SEARCH_CLUSTER_NAME);
+
+    String clusterName = props.nonNullValue(ProcessProperties.CLUSTER_NAME);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Rename "clusterName" which hides the field declared at line 50. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AHiddenFieldCheck)